mycroft/skills/twitter/__init__.py

```diff
@@ -0,0 +1,49 @@
+from os.path import dirname
+
+import tweepy
+
+from adapt.intent import IntentBuilder
+from mycroft.skills.core import MycroftSkill
+from mycroft.util.log import getLogger
+
+__author__ = 'crios'
+
+LOGGER = getLogger(__name__)
+
+
+class TwitterSkill(MycroftSkill):
+    def __init__(self):
+        super(TwitterSkill, self).__init__(name="TwitterSkill")
+        ACCESS_TOKEN = self.config.get('aToken')
+        ACCESS_TOKEN_SECRET = self.config.get('aTokenSecret')
+        CONSUMER_KEY = self.config.get('cKey')
+        CONSUMER_SECRET = self.config.get('cSecret')
+        auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+        auth.set_access_token(ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
+        self.api = tweepy.API(auth)
+
+    def initialize(self):
+        self.load_data_files(dirname(__file__))
+        self.register_regex("that (?P<PostText>.*)")
+        self.register_regex("i am (?P<PostText>.*)")
```

Comment:
This does seem a bit odd. For example, the syntax `Mycroft, post to twitter I am having a bad day` would post `having a bad day`, which may not be what is intended. Also, `Mycroft, post to twitter I found my lost keys` wouldn't trigger the skill at all, even though it seems like it should.


Would it be better/possible to have the regex be any text instead, or at least any text prefixed by `twitter`/`post to twitter`?